crypto/dsa/dsa_lib.c

```diff
@@ -13,15 +13,13 @@
  */
 #include "internal/deprecated.h"
 
-#include <stdio.h>
-#include "internal/cryptlib.h"
-#include "internal/refcount.h"
 #include <openssl/bn.h>
-#include <openssl/asn1.h>
 #include <openssl/engine.h>
-#include "dsa_local.h"
 #include "crypto/dsa.h"
 #include "crypto/dh.h" /* required by DSA_dup_DH() */
+#include "internal/cryptlib.h"
+#include "internal/refcount.h"
```

Comment:
I see that you want to order the inclusions "nicely".  If you observe other places where we've done this sort of ordering, you'll probably see that "internal" comes before "crypto".  The logic is to order them from *furthest away* to *closest*, and that "crypto" is closer to libcrypto than "internal"

Reply:
fixup pushed